wrapanapi/systems/openstack.py

```diff
@@ -1059,3 +1068,92 @@ def usage_and_quota(self):
             'cpu_total': host_cpus,
             'cpu_limit': data['maxTotalCores'] if data['maxTotalCores'] >= 0 else None,
         }
+
+    def list_containers(self):
+        """List of available containers.
+
+        Returns: list of containers name.
+        """
+
+        _, containers = self.sapi.get_account()
+        return [cont.get("name") for cont in containers]
+
+    def create_container(self, container_name):
+        """Create container.
+
+        Args:
+            container_name: name of container
+
+        Return: None
+        """
+
+        self.sapi.put_container(container_name)
+
+    def delete_container(self, container_name):
+        """Delete existing container
+
+        Args:
+            container_name: name of container
+
+        Returns: bool
+        """
+
+        self.sapi.delete_container(container_name)
```

Comment:
I think you should wrap the ClientException that occurs here when the container_name isn't found, and return a specific wrapanapi exception.  `NotFoundError`, `ItemNotFound`, or `VmInstanceNotFound` would be reasonable uses, or you can define a new exception type for `ContainerNotFound`